ui/src/components/chat/edit_group.rs

```diff
@@ -116,312 +105,149 @@ pub fn EditGroup<'a>(cx: Scope<'a, Props<'a>>) -> Element<'a> {
+                                        } else {
+                                            name[..(friend_prefix.len())] == friend_prefix.to_lowercase()
+                                        }
+                                    } ).map(|_friend| {
+
+                                        let friend = _friend.clone();
+                                        rsx!(
+                                            div {
+                                                class: "friend-container",
+                                                aria_label: "Friend Container",
+                                                UserImage {
+                                                    platform: _friend.platform().into(),
+                                                    status: _friend.identity_status().into(),
+                                                    image: _friend.profile_picture()
+                                                },
+                                                div {
+                                                    class: "flex-1",
+                                                    p {
+                                                        aria_label: "friend-username",
+                                                        _friend.username(),
+                                                    },
+                                                },
+                                                Button {
+                                                    aria_label: get_local_text("uplink.remove"),
+                                                    icon: if *edit_group_action.current() == EditGroupAction::Add {
+                                                        Icon::UserPlus
+                                                    } else {
+                                                        Icon::UserMinus
+                                                    },
+                                                    text: if *edit_group_action.current() == EditGroupAction::Add {
+                                                        get_local_text("uplink.add")
+                                                    } else {
+                                                        get_local_text("uplink.remove")
+                                                    },
+                                                    onpress: move |_| {
+                                                        let friend_did = friend.did_key();
+                                                        let mut friends = selected_friends.get().clone();
+                                                        friends.clear();
+                                                        selected_friends.set(friends.clone());
+                                                        friends.insert(friend_did);
+                                                        selected_friends.set(friends);
```

Comment:
great idea, this lets me remove 4 other lines.

had to be `selected_friends.set(vec![friend.did_key()].into_iter().collect());` because it's a hashset